### PR TITLE
Add http_build_query function.

### PR DESCRIPTION
**Pull request by nstdio**
https://www.php.net/manual/en/function.http-build-query.php

### CODE REVIEW

**Comment by nstdio** [comment]
@dim-s is this gonna be merged? Any issue I can address?

**Comment by dim-s** [comment]
@nstdio Can you do answer my code review comments?

**Comment by nstdio** [comment]
I don't see any comments.

**Comment by dim-s** [comment]
Hm, 
image

**Comment by nstdio** [comment]
I guess you need to submit the review

**Comment by dim-s** [comment]
@nstdio I did, see the comments

**Comment by nstdio** [comment]
It appears now, I'll address them, thanks :)

File: exts/jphp-zend-ext/src/main/java/org/develnext/jphp/zend/ext/standard/StringFunctions.java [hunk 1]
```diff
@@ -2957,15 +2949,24 @@ public static String urldecode(String url) throws UnsupportedEncodingException {
         return URLDecoder.decode(url, "UTF-8");
     }
 
-    public static String urlencode(String url) throws UnsupportedEncodingException {
-        return URLEncoder.encode(url, "UTF-8");
+    public static String urlencode(String url) {
+        try {
+            return URLEncoder.encode(url, "UTF-8");
+        } catch (UnsupportedEncodingException e) {
```
[the remark this answers is not in the document]
Comment (nstdio) [reply exts/jphp-zend-ext/src/main/java/org/develnext/jphp/zend/ext/standard/StringFunctions.java]: `UnsupportedEncodingException` cannot be thrown hence we always use `UTF-8` encoding. I find it little bit verbose to handle checked exception witch actually cannot be thrown. WDYT?

File: exts/jphp-zend-ext/src/main/java/org/develnext/jphp/zend/ext/standard/StringFunctions.java [hunk 2]
```diff
@@ -2957,15 +2949,24 @@ public static String urldecode(String url) throws UnsupportedEncodingException {
...
-                .replace("%7E", "~");
+    public static String rawurlencode(String url) {
+        try {
+            return URLEncoder.encode(url, "UTF-8")
+                    // OAuth encodes some characters differently:
+                    .replace("+", "%20")
+                    .replace("*", "%2A")
+                    .replace("%7E", "~");
+        } catch (UnsupportedEncodingException e) {
+            throw new RuntimeException(e);
```
[the remark this answers is not in the document]
Comment (nstdio) [reply exts/jphp-zend-ext/src/main/java/org/develnext/jphp/zend/ext/standard/StringFunctions.java]: And the same is here.

**Pull request merged**